lib/formatters/stringFormatter.js

```diff
@@ -22,13 +22,18 @@ const levelColors = {
  */
 function deprecationsFormatter(results) {
 	const allDeprecationWarnings = results.flatMap((result) => result.deprecations);
-	const uniqueDeprecationWarnings = _.uniqBy(allDeprecationWarnings, 'text');
 
-	if (!uniqueDeprecationWarnings || !uniqueDeprecationWarnings.length) {
+	if (!allDeprecationWarnings || !allDeprecationWarnings.length) {
```

Comment:
[suggestion] How about refactoring the code as below?

```suggestion
	if (allDeprecationWarnings.length === 0) {
```